docker/package/model.py

```diff
@@ -284,6 +290,17 @@ def gen_postinst(self, out):
         with open(out, 'w') as f:
             f.write(postinst_contents)
 
+    def gen_postrm(self, out):
+        postinst_contents = f'''#!/bin/sh
+
+set -e
+
+#DEBHELPER#
+
+{self.postrm_steps}
+'''
+        with open(out, 'w') as f:
+            f.write(postinst_contents)
```

Comment:
I guess it doesn't really make a difference, but shouldn't this be `postrm_contents` instead (same just above)?